src/tmpfiles/tmpfiles.c

```diff
@@ -1614,8 +1615,12 @@ static int create_directory_or_subvolume(const char *path, mode_t mode, bool sub
                 return pfd;
 
         if (subvol) {
-                if (btrfs_is_subvol(empty_to_root(arg_root)) <= 0)
-
+                r = getenv_bool("SYSTEMD_TMPFILES_FORCE_SUBVOL");
+                if (r < 0) {
+                      log_warning_errno(r, "Cannot parse value of $SYSTEMD_TMPFILES_FORCE_SUBVOL, ignoring.");
```

Comment:
change to:

```c
if (r < 0) {
        if (r != -ENXIO) /* env var not set */
                log_warning_errno(r, "Cannot parse…");
        r = btrfs_is_subvol(…
}
…
```

i.e. filter out the ENXIO case explicitly, which is what getenv_bool() returns if the env var is not set